Mathlib/Analysis/InnerProductSpace/Projection.lean

```diff
@@ -586,6 +587,19 @@ variable (K)
 theorem orthogonalProjection_norm_le : ‖K.orthogonalProjection‖ ≤ 1 :=
   LinearMap.mkContinuous_norm_le _ (by norm_num) _
 
+/-- The orthogonal projection onto a closed subspace is norm non-increasing. -/
+theorem norm_orthogonalProjection_le (v : E) :
+    ‖orthogonalProjection K v‖ ≤ ‖v‖ := by calc
```

Comment:
The first of these should be called `norm_orthgonalProjection_le` and the one you're adding should be called `norm_orthogonalProjection_apply`. We should probably also have a version `norm_orthgonalProjection (hK : K ≠ ⊥) : ‖K.orthogonalProjection‖ = 1`.